### PR TITLE
fix migrations on fresh database

### PR DESCRIPTION
**Pull request by Akasch**
the migration did only work on existing databases where the
ContentType table was already exising while executing this
migrations. Now it is working with and without existing
database.

For installations that have executed only the old migration 0002
the migration 0003 does nothing and the migration 0004 just
alter the fields to the definition they already are using
so nothing happens

### CODE REVIEW

**Comment by Akasch** [comment]
once this is merged i would bump the version to 0.2.2

File: django_token_bucket/migrations/0003_auto_20200111_1625.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from django.contrib.auth import get_user_model
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    token_bucket_model = apps.get_model('django_token_bucket', 'TokenBucket')
+    content_type_model = apps.get_model('contenttypes', 'ContentType')
+    for row in token_bucket_model.objects.all():
+        if not row.content_type:
+            row.content_type = content_type_model.objects.get_for_model(get_user_model())
```
Comment (wichmannpas) [inline django_token_bucket/migrations/0003_auto_20200111_1625.py]: To be safe, migrations should explicitly name the database to use (here `using(db_alias)`, where `db_alias` is  `schema_editor.connection.alias`).

File: django_token_bucket/migrations/0003_auto_20200111_1625.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.1.15 on 2020-01-11 15:25
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    token_bucket_model = apps.get_model('django_token_bucket', 'TokenBucket')
+    content_type_model = apps.get_model('contenttypes', 'ContentType')
+    for row in token_bucket_model.objects.all():
```
Comment (wichmannpas) [inline django_token_bucket/migrations/0003_auto_20200111_1625.py:12]: To be safe, migrations should explicitly name the database to use (here `objects.using(db_alias)`, where `db_alias` is  `schema_editor.connection.alias`).

**Review by wichmannpas — APPROVED** [review]
A minor comment, can be merged when fixed.

**Review by wichmannpas — APPROVED** [review]

**Pull request merged**